### DISCUSSION TITLE
How to open API Port 2375 for external connects.

### DISCUSSION THREAD

**Discussion by flobert8** [louislam/dockge#611] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I want to allow external applications to connect to the docker instance. For example my homepage dashboard.
For that i need an open port 2375 to connect from external applications.
Whats the best way to do that? I have installed dockge over an Proxmox Helper Script within an LXC.
Hope someone could help.

BR

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

LXC

### 🌐 Browser

Firefox

### 🐋 Docker Version

27.2.1

**Comment by dimatx** [comment]
See method 2 section of https://github.com/louislam/uptime-kuma/wiki/How-to-Monitor-Docker-Containers

You'll likely need to do the part about removing "-H fd://" from the ExecStart. Also discussed in that section.
